### PR TITLE
Removing Publishing Steps from public build

### PR DESCRIPTION
**Pull request by embetten**
- Remove publishing symbols and nupkg package from public build to internal release pipeline

### CODE REVIEW

File: build/build.yml [hunk 1]
```diff
@@ -178,20 +178,6 @@ steps:
...
-      PublishSymbols: true
-      SymbolServerType: TeamServices
-      SymbolsProduct: 'artifacts-credprovider'
-    env:
-      ARTIFACTSERVICES_SYMBOL_ACCOUNTNAME: 'microsoft'
-      ARTIFACTSERVICES_SYMBOL_USEAAD: 'false'
-
 - ${{ if eq(parameters.publish, 'true') }}:
   - task: PublishPipelineArtifact@1
     displayName: 'Upload symbols'
```
Comment (JohnSchmeichel) [inline build/build.yml:183]: If the pipeline artifact has symbol files included in it would say also remove this one then?

Comment (JohnSchmeichel) [reply build/build.yml:183]: Then I think all the parameters.publish usage is gone and we can remove that from all the other yamls

Comment (embetten) [reply build/build.yml]: good point removed symbols upload and removed unused publish parameter

**Comment by JohnSchmeichel** [comment]
.vsts-release.yml should be obsolete at this point? can we remove it now too?

**Comment by JohnSchmeichel** [comment]
sorry one more: there is usage of PushNupkg which is obsolete across the GitHub yamls?

**Review by JohnSchmeichel — APPROVED** [review]

**Pull request merged**